.github/workflows/ecosystem-ci-trigger.yml

```
@@ -22,44 +46,123 @@ jobs:
                 repo: context.repo.repo,
                 username: user,
               });
-              hasTriagePermission = data.user.permissions.triage
+              hasTriagePermission = ['triage', 'write', 'admin'].includes(data.permission);
```

shouldn't this check data.user.permissions ? also would have to look up what data.user.permissions is exactly but i assume a map of boolean values
```
{
  triage: true,
  admin: false
}
```

so this could work

```suggestion
              hasTriagePermission = ['triage', 'write', 'admin'].some(p => data.user.permissions[p]);
```